newrelic.md

```diff
@@ -27,6 +27,28 @@ To view data about application performance, end user experience, and server perf
 3. After being redirected via SSO to your account(production or staging app) on the New Relic site.
 4. Click on the application, it will take you to the Dashboard(APM) for Monitoring.
 
+## App apdex score
```

Comment:
@dev-sandeepsingh :  Here can we keep it as steps to follow for deployment and configuring various services/add-ons as required? Knowing the concept while configuring something is good, so can we just add a link to that information instead of adding that information itself (in case someone who wants to know more can follow the link)?
What do you think?